lxc/copy.go

```diff
@@ -304,6 +321,47 @@ func (c *cmdCopy) copyContainer(conf *config.Config, sourceResource string,
 			dest = dest.UseTarget(c.flagTarget)
 		}
 
+		if c.flagRefresh {
+			// Make sure the target container exists if using --refresh
+			container, etag, err := dest.GetContainer(destName)
+			if err != nil {
+				return fmt.Errorf("Failed to refresh target container '%s': %v",
+					destName, err)
+			}
+
+			// Target container needs to be stopped
+			if container.Status == "Running" {
+				return fmt.Errorf("Target container needs to be stopped when using --refresh")
```

Comment:
As mentioned above, this should be an API error coming from the target server rather than something we check in the CLI tool.